qa/rpc-tests/bip65-cltv-p2p.py

```diff
@@ -26,7 +26,11 @@ class BIP65Test(ComparisonTestFramework):
 
     def setup_network(self):
         self.nodes = start_nodes(self.num_nodes, self.options.tmpdir,
-                                 extra_args=[['-debug', '-whitelist=127.0.0.1']],
+                                 extra_args=[[
+                                    '-debug',
+                                    '-whitelist=127.0.0.1',
+                                    '-allowdeprecated=getnewaddress',
+                                ]],
```

Comment:
```suggestion
                                 ]],
```

Reply:
Doesn't this also require `-experimentalfeatures`? Or is that always set by `start_nodes`? (Similarly for other cases below.)

Reply:
:+1: